components/landing/Landing.vue

```diff
@@ -68,12 +68,12 @@
           </div>
         </div>
       </div>
-      <div v-if="prefix === 'rmrk' || prefix === 'bsx'">
-        <LazyCarouselLatestSales class="my-5" />
-        <span v-if="prefix === 'rmrk'">
-          <LazyCarouselPopularCollections class="my-5" />
-        </span>
-        <LazyCarouselNewestList class="my-5" />
+      <div v-if="prefix === 'rmrk' || prefix === 'bsx'" class="py-5">
+        <LazyCarouselTypeLatestSales class="my-5" />
+        <LazyCarouselTypePopularCollection
+          v-if="prefix === 'rmrk'"
+          class="my-5" />
+        <LazyCarouselTypeNewestList class="my-5" />
```

Comment:
😕

Reply:
> CarouselIndex will necessarily have a title & a subtitle, I think it's a better to put it directly in the component wrapper

hi @roiLeo, can you elaborate more on this? did you mean we change `<LazyCarouselType />` with `<CarouselIndex />` instead?

> why do you need a 'module-old' folder?

because we have redesigned components in the future, I put the existing design into `module-old` and the new one in `module`. to make it easier to separate old and new design